pub/index.html

```diff
@@ -0,0 +1,168 @@
+
+      }
+      .fixed p {
+        margin: 0;
+      }
+      footer {
+        clear: both;
+        position: relative;
+        text-align: center;
+      }
+      footer img {
+        width: 35px;
+      }
+
+
+    </style>
+  </head>
+<body>
+  <header></header>
+  <div class="fixed"> <p>+ dat fixed div | bed time</p></div>
+  <div class="container">
+
+  <aside>
+  <div class="logodiv"></div>
+  <h3>interesting | aside</h3>
+  <p>not really</p>
+  <hr>
+  <p>according to what my HTML.md file says an aside is supposed to contain
+    information that is a tangent of the information on the page ...weird</p>
+  <p>more shenanigans</p>
+  <p><a href="http://www.roomstogo.com/furniture/Bedrooms/Queen-Beds/_/N-8dd">so tired</a></p>
+      <p><a href="http://www.overstock.com/Bedding-Bath/Pillows/4547/cat.html">down!</a></p>
+  </aside>
+  <article>
+    <h1>She Works Hard for the Money</h1>
+    <p class="artist">by Donna Summer</p>
+    <p>She works hard for the money
+      So hard for it, honey
+      She works hard for the money
+      So you better treat her right</p>
+      <p>She works hard for the money
+        So hard for it, honey
+        She works hard for the money
+        So you better treat her right</p>
+        <p>I met her there in the corner stand
+          And wonders where she is
+          And it's strange to her
+          Some people seem to have everything
+          9 am on the hour hand
+          And she's waiting for the bell
+          And she's looking real pretty
+          She's waiting for her clientele</p>
+          <img src="http://www.importantstudio.com/ironyard/cartmanfancy.jpg" alt="fancy eric">
+          <p>She works hard for the money
+            So hard for it, honey
+            She works hard for the money
+            So you better treat her right</p>
+            <p>She works hard for the money
+              So hard for it, honey
+              She works hard for the money
+              So you better treat her right</p>
+
+    </article>
+
+  <article>
+    <h1>Stinky Britches</h1>
+    <p class="artist">by South Park</p>
+    <p>Stinky britches, you've got those stinky britches.
+      Stinky britches, you've got them stinky britches.
+      Stinky britches, you've got stinky britches.
+      Whew! Stinky!</p>
+      <img src="http://www.importantstudio.com/ironyard/chef.jpg" alt="chef">
+
+
+  </article>
+  <article>
+    <h1>O Canada!</h1>
+    <p class="artist">by some canadian</p>
+    <p>O Canada!
+      Our home and native land!
+      True patriot love in all thy sons command.
+      With glowing hearts we see thee rise,
+      The True North strong and free!
+      From far and wide,
+      O Canada, we stand on guard for thee.
+      God keep our land glorious and free!
+      O Canada, we stand on guard for thee.
+      O Canada, we stand on guard for thee.</p>
+      <img src="http://www.importantstudio.com/ironyard/canadian.png" alt="canadians">
+  </article>
```

Comment:
So did that come out the way you expected? Gotta find some Cartman Ipsum for your next iteration...